store/tikv/backoff.go

```diff
@@ -218,8 +218,8 @@ const (
 	scatterRegionBackoff           = 20000
 	waitScatterRegionFinishBackoff = 120000
 	locateRegionMaxBackoff         = 20000
-	pessimisticLockMaxBackoff      = 10000
```

Comment:
also need pay attention and test to take look `WithVars` https://github.com/pingcap/tidb/blob/d20195a3a5205bce876afa02f0954d682476918f/store/tikv/txn.go#L385
will make real max be `pessimisticLockMaxBackoff x 2` or maybe in this situation we should ignore `BackOffWeight`? @jackysp

Reply:
Transaction related `NewBackoffer` call are mostly  used with `WithVars` and will be affected by `BackOffWeight`,  this change will make pessimistic max backoff time doubled as before. Are there any corner cases using `BackOffWeight` ? @jackysp

Reply:
10s is too short compares to other maxBackoff, I think it's doesn't hurt to increase to 20s

Reply:
We could discuss it this afternoon.